Makefile

```diff
@@ -1,26 +1,27 @@
 CXX=g++
-CXXFLAGS=-std=c++11 -I. -Wall -Werror
+CXXFLAGS=-std=c++11 -I. -Wall -Werror -static-libgcc -static-libstdc++ -pthread -Wl,-Bstatic
+LDLIBS= -lboost_log_setup -lboost_log -lboost_thread -lboost_regex -lboost_system
 OPTIMIZE=-O2
-BOOSTFLAG = -DBOOST_LOG_DYN_LINK -lboost_system -lboost_regex -lboost_log -lboost_thread -lpthread -lboost_log_setup
 DEPS=server.h connection.h config_parser.h request.h response.h request_handler.h echo_handler.h static_handler.h not_found_handler.h status_handler.h reverse_proxy_handler.h http_client.h sleep_handler.h
 OBJ=server.o connection.o config_parser.o request.o response.o request_handler.o echo_handler.o static_handler.o not_found_handler.o status_handler.o reverse_proxy_handler.o http_client.o sleep_handler.o
 GTEST_DIR=googletest/googletest
 TESTS=config_parser_test connection_test server_test request_test echo_handler_test static_handler_test not_found_handler_test request_handler_test status_handler_test reverse_proxy_handler_test http_client_test response_test
+DEPLOYS=deploy binary.tar webserver-image
 
 default: webserver
 
 %.o: %.cc $(DEPS)
-	$(CXX) $(CXXFLAGS) $(OPTIMIZE) $(COV) -c -o $@ $< $(BOOSTFLAG)
+	$(CXX) $(CXXFLAGS) $(OPTIMIZE) $(COV) -c -o $@ $<
 
 webserver: main.cc $(OBJ)
-	$(CXX) $(CXXFLAGS) $(OPTIMIZE) $(COV) -o $@ $^ $(BOOSTFLAG)
+	$(CXX) $(CXXFLAGS) $(OPTIMIZE) $(COV) -o $@ $^ $(LDLIBS)
```

Comment:
Not really sure tbh... Spec seemed to indicate to just change it, but no idea if that was just vague.